flake.nix

```diff
@@ -10,5 +10,9 @@
     pkgs = import nixpkgs { inherit system; };
   in {
     devShells.default = import ./UI { inherit pkgs; };
+    apps.default = {
+      type = "app";
+      program = "${pkgs.ladybird}/bin/Ladybird";
```

Comment:
I would expect `github:LadybirdBrowser/ladybird` to run the respective version, not whatever is currently packaged in whatever `nixpkgs` version we pin.

Reply:
Oh you're right... Don't know what I was thinking...
Will fix that